tests/test_cuda_bd_exchange_mover.py

```diff
@@ -271,21 +271,20 @@ def test_bd_exchange_deterministic_moves(moves, precision, seed):
     if precision == np.float64:
         klass = custom_ops.BDExchangeMove_f64
 
-    # Reference that makes a single proposal per move
-    bdem_a = klass(N, group_idxs, params, DEFAULT_TEMP, nb.potential.beta, cutoff, seed, 1, 1)
+    # Reference version
+    bdem_a = klass(N, group_idxs, params, DEFAULT_TEMP, nb.potential.beta, cutoff, seed, proposals_per_move, 1)
```

Comment:
Should `proposals_per_move` be 1 here? (confused why `bdem_a` and `bdem_b` look identical)

Reply:
They are identical, refer to this comment https://github.com/proteneer/timemachine/pull/1234#discussion_r1454114920

Only verifying that given the same setup you produce the same result. Right now we are focusing on the bitwise determinism of batch vs serial rather than proposals per move

Reply:
Ah, is this only testing determinism? Thought it might also be intended to test consistency with different numbers of `proposals_per_move` (hadn't yet comprehended the comment below).

Reply:
OK, disregard my last comment (hadn't seen yours yet)

Reply:
Yes, previously we ensured determinism between movers with different `proposals_per_step` which requires a more complicated approach to generating noise. Talking to @proteneer it seems best to focus on bitwise determinism of the serial vs batch where the number of proposals in a single move call is identical, rather than the case this was previously testing.

Reply:
> it seems best to focus on bitwise determinism of the serial vs batch where the number of proposals in a single move call is identical

Got it. To clarify, does ensuring bitwise-identical results for different values of `proposals_per_move` add significant complexity? (It does seem useful at least as a way to test the batched implementation)

Reply:
(marking resolved; follow-up discussion in this thread: https://github.com/proteneer/timemachine/pull/1234#discussion_r1454114920)